from app.errors.base_error import BaseError


class ResourceNotFoundError(BaseError):
    pass
